onedal/datatypes/table.cpp

```diff
@@ -72,6 +72,10 @@ ONEDAL_PY_INIT_MODULE(table) {
         const auto column_count = t.get_column_count();
         return py::make_tuple(row_count, column_count);
     });
+    table_obj.def_property_readonly("dtype", [](const table& t){
```

Comment:
```suggestion
    table_obj.def_property_readonly("get_numpy_dtype", [](const table& t){
```
I suggest to use this or just `numpy_dtype` naming, just not to be confused

Reply:
And again, from my point of view for tables API, this seems unnecessary and looks like a workaround or a hack.